src/activities.js

```diff
@@ -0,0 +1,85 @@
+import ToDoList from './mainFunctions.js';
+import TrashButton from './Trash.jpg';
+
+export const addItem = (data) => {
+  const newListItem = new ToDoList(data);
+  localStorage.setItem('todoList', JSON.stringify(newListItem.getList()));
+};
+
+export const clear = (index) => {
+  ToDoList.list = ToDoList.list.filter((item) => item !== ToDoList.list[index]);
+  ToDoList.list.forEach((item, i) => {
+    item.index = i;
+  });
+  localStorage.setItem('todoList', JSON.stringify(ToDoList.list));
+};
+
+export const update = (index, text) => {
+  ToDoList.list[index].description = text;
+  localStorage.setItem('todoList', JSON.stringify(ToDoList.list));
+};
+
+export const displayList = () => {
```

Comment:
- [ ] For this specific project there should be an array with different tasks (example below) so that on opening the app those tasks are dynamically rendered on the page using a function. Kindly implement that.

  ```
  const tasks = [
    {
      description: 'First task',
      completed: false,
      index: 1,
    },
    {
      description: 'Second task',
      completed: true,
      index: 2,
    }
  ];
  ```